actors/util/adt/balancetable.go

```diff
@@ -55,9 +62,23 @@ func (t *BalanceTable) Add(key addr.Address, value abi.TokenAmount) error {
 	return (*Map)(t).Put(AddrKey(key), &sum)
 }
 
+// Adds an amount to a balance. Create entry if not exists
+func (t *BalanceTable) AddCreate(key addr.Address, value abi.TokenAmount) error {
+	prev, err := t.Get(key)
+	if err != nil {
+		_, nf := err.(ErrNotFound)
```

Comment:
Avoid the sentinel error logic by using `(*Map)(t).Get(AddrKey(key), &value)` here.